### PR TITLE
New Instrument instructions

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Potentially addresses #793 by updating and revising the new instrument example in the docs.

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update

# How Has This Been Tested?

CI tests.

**Test Configuration**:
* Operating system: ReadTheDocs, GitHub Actions
* Version number: Python 3.9
* Any details about your local setup that are relevant

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the release
checklist on the wiki: https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

File: docs/new_instrument.rst [hunk 1]
```diff
@@ -401,20 +422,23 @@ The load module method signature should appear as:
...
   object is generally trivial. If this isn't the case, it can be tedious to
   fill out all information if there are many data parameters. In this case it
-  may be easier to fill out a text file. A basic convenience function is
-  provided for this situation. See :py:meth:`pysat.Meta.from_csv` for more
-  information.
+  may be easier to create a text file, though in many cases a separate function
+  is defined to provide metadata for specific data types (see
+  :py:func:`pysatSpaceWeather.instruments.methods.kp_ap.initialize_kp_metadata`).
+  A basic convenience function is provided if you decide to use a text file.
+  See :py:meth:`pysat.Meta.from_csv` for more information.
```
Comment (aburrell) [inline docs/new_instrument.rst:436]: @rstoneback can you provide an example of when this is used?

File: docs/new_instrument.rst [hunk 2]
```diff
@@ -763,6 +788,27 @@ Note that :py:mod:`pysat` assumes that this flag is True if no variable is
 present. Thus, specifying only ``_test_download = {'': {'Level_1': False}}``
 has the same effect, and Level 2 tests will still be run.
 
+Load Options
```
Comment (aburrell) [inline docs/new_instrument.rst:791]: @jklenzing I think I included the missing test options.  Can you verify that?

**Review by aburrell — COMMENTED** [review]
An appropriate line already exists in the changelog, so I just checked its existing format.

File: docs/new_instrument.rst [hunk 3]
```diff
@@ -813,17 +864,3 @@ present.
...
-.. code:: Python
-
-    def init(self):
-        """Initializes the Instrument object with instrument specific values."""
-
-        self.acknowledgements = acknowledgements_string
-        self.references = references_string
-        pysat.logger.info(self.acknowledgements)
-
-        return
```
Comment (jklenzing) [inline docs/new_instrument.rst:829]: why is this deleted?

File: docs/new_instrument.rst [hunk 4]
```diff
@@ -813,17 +864,3 @@ present.
 
 Data Acknowledgements
 ---------------------
```
Comment (jklenzing) [inline docs/new_instrument.rst:866]: this looks like an orphan header

File: docs/new_instrument.rst (hunk 2 above)
Comment (jklenzing) [reply docs/new_instrument.rst:796]: this looks good to me.

**Review by jklenzing — APPROVED** [review]

File: docs/new_instrument.rst (hunk 1 above)
Comment (aburrell) [reply docs/new_instrument.rst:461]: Currently addressed by opening #1113
